webapp/graphite/storage.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, finders=None, hosts=None):
 
     if hosts is None:
       hosts = settings.CLUSTER_SERVERS
-    remote_hosts = [host for host in hosts if not is_local_interface(host)]
+    remote_hosts = [host for host in hosts if not settings.REMOTE_EXCLUDE_LOCAL or not is_local_interface(host)]
```

Comment:
It is different than the current default behavior, which is intended. Not sure how best to handle communicating that change. Perhaps make it default on now with a comment that the we intend to reverse the default behavior in a future release? Honestly I think the change will affect very few people, and the current default behavior seems questionably implemented at best.